java/arcs/core/data/InformationFlowLabel.kt

```diff
@@ -42,8 +42,31 @@ sealed class InformationFlowLabel {
             override fun toString() = "($lhs and $rhs)"
         }
 
-        infix fun and(other: Predicate) = Predicate.And(this, other)
-        infix fun or(other: Predicate) = Predicate.Or(this, other)
-        fun not() = Predicate.Not(this)
+        infix fun and(other: Predicate) = And(this, other)
+        infix fun or(other: Predicate) = Or(this, other)
+        fun not() = Not(this)
+
+        companion object {
+            /** Combines a list of >= 2 predicates via the [And] operator. */
+            fun and(predicates: List<Predicate>): And = combine(predicates) { a, b -> a and b }
```

Comment:
wouldn't this one be more ergonomic with an vararg?

Instead of `redicate.and(listOf(...` you will be able to do just `Predicate.and(...` ?

Reply:
Good idea, done!